classes/options-general.php

```diff
@@ -692,7 +692,7 @@ class='howto'><?php _e( 'HTML5 - more precise, may prompt users for permission,
 
 											<p class="submit">
 												<input type="submit" class="button-primary"
-												       value="<?php _e( 'Save Options', 'simplemap' ) ?>"/><br/><br/>
+												       value="<?php _e( 'Save Options', 'SimpleMap' ) ?>"/><br/><br/>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->